dotty-tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/api/compat/v1/Operation.scala

```diff
@@ -0,0 +1,6 @@
+package me.shadaj.scalapy.tensorflow.api.compat.v1
```

Comment:
Hmm do we want to test our `compat.v1` APIs against TF v1 in CI? I don't think at this point this is really guaranteed to work against TF v1... keeping `compat.v1` that aren't tested can be misleading for the prospective users :/

Reply:
thats the name of the package in TF2 we decided to have the same package structure. Additionally TF2 uses `Operation` form TF1.0 so I don't see it as a flaw in our design but in theirs.

Reply:
Oh ok :facepalm: ;D

Reply:
It's kinda still the same question about whether we want to be more aligned with the underlying Python APIs (at the expense of less clarity), or have a clear API (at the expense of the API less corresponding to the Python API)... but yeah, maybe diverging too far from Python is gonna be confusing for the users coming from Python

Reply:
I think they keep it that way so people know it's the same as in previous major version, nevertheless it's confusing enough as it is. Moving it would only increase complexity